test/distributed/org/apache/cassandra/distributed/test/InternodeEncryptionEnforcementTest.java

```diff
@@ -86,8 +84,7 @@ public void testInboundConnectionsAreRejectedWhenAuthFails() throws IOException,
             // Wait for authentication to fail
             cluster.get(1).logs().watchFor("Unable to authenticate peer");
             cluster.get(1).runOnInstance(runnable);
-            cluster.get(2).logs().watchFor("Unable to authenticate peer");
-            cluster.get(2).runOnInstance(runnable);
```

Comment:
nit: how about adding a watch for "Connection reset by peer" in instances 2's log?

```
            cluster.get(2).logs().watchFor("Connection reset by peer");
```

Reply:
I think we should watch for "Unable to authenticate peer" which is more specific in terms of why the connection reset happen. Also it is the log that we print after `authenticate` call fails.

Reply:
ok.